src/brain/issueLabelHandler/index.test.ts

```diff
@@ -797,7 +877,7 @@ describe('issueLabelHandler', function () {
       }
     );
 
-    it.only.each([
```

Comment:
😬

Reply:
?

Reply:
We were skipping tests, I must've accidentally committed this while running tests locally